cmake/OpenCVModule.cmake

```diff
@@ -879,6 +878,8 @@ macro(_ocv_create_module)
     COMPILE_PDB_OUTPUT_DIRECTORY ${LIBRARY_OUTPUT_PATH}
     LIBRARY_OUTPUT_DIRECTORY ${LIBRARY_OUTPUT_PATH}
     RUNTIME_OUTPUT_DIRECTORY ${EXECUTABLE_OUTPUT_PATH}
+    DEFINE_SYMBOL CVAPI_EXPORTS
+    POSITION_INDEPENDENT_CODE TRUE
```

Comment:
"PIC" is needed for Python/Java bindings only.
For using OpenCV with applications it is not required at all.

Reply:
This is a port of block removed from _OpenCVCompilerOptions.cmake_. We always enable PIC because static libraries can be used to build some shared libraries, I wanted to preserve this ability.